sdk-libs/macros/docs/features/comparison.md

```
@@ -0,0 +1,259 @@
+| `Account<'info, T>` | `Account<'info, T>` (with RentFree derive) |
+| `#[account(init)]` | `#[account(init)]` + compression hooks |
+| `#[account(init, token::mint = m)]` | `#[rentfree_token]` + `#[light_mint]` |
+| `Program<'info, Token>` | `Program<'info, CompressedToken>` |
+| `Account<'info, Mint>` | `UncheckedAccount<'info>` + `#[light_mint]` |
+| `Account<'info, TokenAccount>` | `Account<'info, T>` with `#[rentfree_token]` |
+| `close = destination` | `close = destination` + compression cleanup |
+| Manual token CPI | Auto-generated in `light_pre_init()` |
+
+### Type Mapping
+
+| Anchor SPL Type | Light RentFree Type |
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)